robot/resources/lib/python_keywords/object_access.py

```diff
@@ -142,6 +143,12 @@ def can_get_head_object(
     return True
 
 
+def _generate_random_range_cut(start: int = 0, end: int = 10):
+    start = random.randint(start, end)
+    end = random.randint(start, end)
+    return f"{start}:{end}"
```

Comment:
that is not `[start:end]` meaning but is `[offset:height]` in fact. so i would like to ask you to rename it accordingly to prevent any misunderstanding. also, we do not allow `[X:0]` requests do we have any tests for it? just to fix it

btw:
```
▶ neofs-cli object range -h | grep '\-\-range'
      --range string          Range to take data from in the form offset:length
```

so we are not that bad and try to doc it (i can imagine we would have written it as just "Range to take data from"). but i agree that slicing format (`[start:end]`) may be more presentative

Reply:
for additional tests - https://github.com/nspcc-dev/neofs-testcases/issues/652
fixed naming and the code accordingly